src/containers/jurisdictions/JurisdictionLabel.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import { N2, PP4 } from '../../core/style/Colors';
+import {
+  MEDIA_QUERY_JUR_SM,
+  MEDIA_QUERY_JUR_MD,
+  MEDIA_QUERY_JUR_LG
+} from '../../core/style/Sizes';
+
+const LabelWrapper = styled.span`
+  display: flex;
+  flex-basis: 4%;
+  flex-direction: column;
+  font-size: 12px;
+  margin: 0 10px 20px 10px;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_SM}px) {
+    flex-basis: 10%;
+    font-size: 14px;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
+    flex-basis: 20%;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_LG}px) {
+    position: absolute;
+    left: ${props => `${props.x}px`};
+    margin: 0;
+    top: ${props => `${props.y}px`};
+  }
+`;
+
+const LocationLabel = styled.span`
+  color: ${PP4};
+  font-weight: normal;
+  margin-bottom: 4px;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
+    font-weight: 600;
+  }
+`;
+
+const PopulationLabel = styled.span`
+  color: ${N2};
+  display: none;
+  font-weight: normal;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
```

Comment:
The population label disappears when the screen drops below 900px.

![screen shot 2018-06-25 at 10 05 57 am](https://user-images.githubusercontent.com/440299/41864639-8680a41a-785f-11e8-8260-06ec5cda10f9.png)

Reply:
Fixed